sale_order_line_chained_move/models/stock_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class StockMove(models.Model):
+
+    _inherit = "stock.move"
+
+    related_sale_line_id = fields.Many2one(comodel_name="sale.order.line", index=True,)
```

Comment:
Hi @rousseldenis. the index will generate extra time when writing on ``stock.move``. (if I understand correctly indexes ...) Do you think it's necessary ?

Reply:
Do you have metrics to compare gain with/without indexes on such records that cannot be computed without recursive function otherwise?

The index value is inserted on the same query as normal flow (see https://github.com/OCA/sale-workflow/pull/1564/files#diff-35bc9f8bfb82e6c1af01cbf043eb5fc0b52a3397bb2f083ee0fabafc9664adb1R31). So, I think this addition should not influence write performances.

Reply:
> Do you have metrics to compare gain with/without indexes

No I don't. I just know that it is not allways a good idea to create indexes on fields, because it can make insertion / update slower. And so it should be reserved for very searched fields. (name, company_id, etc...)
But maybe I'm wrong !